.github/workflows/auth-tests.yml

```diff
@@ -22,18 +22,17 @@ jobs:
     strategy:
       fail-fast: true
     steps:
-      - name: Azure CLI login
-        run: az login --allow-no-subscriptions --output none --service-principal --tenant "${TENANT_ID}" --username "${CLIENT_ID}" --password "${CLIENT_SECRET}"
+      - run: az login --allow-no-subscriptions --output none --service-principal --tenant "${TENANT_ID}" --username "${CLIENT_ID}" --password "${CLIENT_SECRET}"
 
-      - name: Install Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: 1.17.6
+      - uses: actions/checkout@v2
```

Comment:
it'd be worth making this `v3` if possible, `v2`'s been unreliable of late

Reply:
Good catch, thanks 👍